### PR TITLE
Make turning on/off the light's EBC configurable.

### PR DESCRIPTION
**Pull request by lucasb-eyer**
Accept this if you want @gestom to leave the next meeting healthy.

### CODE REVIEW

File: scitos_docking/src/scitos_docking/CChargingActions.cpp [hunk 1]
```diff
@@ -23,6 +23,8 @@ CChargingActions::CChargingActions(ros::NodeHandle *n)
 	warningLevel = .2;
 	poseSet = true;
 	injectX = injectY = injectPhi = 0;
+
+	nh->getParam("lightEBC", light.ebcport);
 }
```
Comment (marc-hanheide) [inline scitos_docking/src/scitos_docking/CChargingActions.cpp:28]: May I ask what this line does @lucasb-eyer? You're getting the param but doing nothing with it??? Why??

**Comment by marc-hanheide** [comment]
Besides that one comment, this looks good to me... I'm :+1: to merge it. If @gestom doesn't protest until tomorrow, I'll merge it.

**Comment by gestom** [comment]
looks OK

File: scitos_docking/src/scitos_docking/CChargingActions.cpp (hunk 1 above)
Comment (lucasb-eyer) [inline scitos_docking/src/scitos_docking/CChargingActions.cpp:28]: I'm storing it in `light.ebcport`, which is being used by `light` itself in its `set` method further down. I didn't bother making it private+getter+setter.

Comment (marc-hanheide) [inline scitos_docking/src/scitos_docking/CChargingActions.cpp:28]: Oh... It was late... Nevermind. Sorry.

**Pull request merged**